build.gradle

```diff
@@ -258,6 +258,16 @@ dependencyUpdates.resolutionStrategy = {
 
 gradle.startParameter.continueOnFailure = true
 
+// during check task:
```

Comment:
This should not be here ...

Reply:
Think I copied this but anyway, will remove.